language/src/main/java/de/monticore/lang/sysmlv2/SysMLv2Mill.java

```diff
@@ -291,4 +293,42 @@ protected FunctionSymbol buildMessagesFunction(TypeSymbol streamSymbol,
         .setSpannedScope(new BasicSymbolsScope())
         .build();
   }
+
+protected FunctionSymbol buildTakesFunction(TypeSymbol streamSymbol, TypeVarSymbol typeVar) {
```

Comment:
```suggestion
  protected FunctionSymbol buildTakesFunction(TypeSymbol streamSymbol, TypeVarSymbol typeVar) {
```

Ausserdem die Zeilenlänge (max. 80 Zeichen) beachten